src/main/java/bio/terra/service/filedata/azure/util/BlobContainerClientFactory.java

```diff
@@ -0,0 +1,201 @@
+package bio.terra.service.filedata.azure.util;
+
+import com.azure.core.credential.AzureSasCredential;
+import com.azure.core.credential.TokenCredential;
+import com.azure.core.http.HttpClient;
+import com.azure.storage.blob.*;
+import com.azure.storage.blob.models.UserDelegationKey;
+import com.azure.storage.blob.sas.BlobSasPermission;
+import com.azure.storage.blob.sas.BlobServiceSasSignatureValues;
+import com.azure.storage.common.StorageSharedKeyCredential;
+import com.azure.storage.common.sas.SasProtocol;
+import java.time.OffsetDateTime;
+import java.time.ZoneOffset;
+import java.util.Locale;
+import java.util.Objects;
+import org.apache.commons.lang.StringUtils;
```

Comment:
If you use the commons library, please use `lang3`, which is a newer version than `lang`.
```suggestion
import org.apache.commons.lang3.StringUtils;
```

Reply:
Updated in the last commit.